src/api/data/cache/cache.go

```diff
@@ -97,10 +93,16 @@ func (c *cachedCharts) All() ([]*models.ChartPackage, error) {
 	c.rwm.RLock()
 	defer c.rwm.RUnlock()
 	var allCharts []*models.ChartPackage
+	reposCollection, err := data.GetRepos()
+	if err != nil {
+		return nil, err
+	}
+	repos := []*data.Repo{}
```

Comment:
I personally prefer to use:

```
var repos []*data.Repo
```

in this case, since it better conveys the idea that it will get filled by `FindAll`